cypress/integration/BounceRulesContainer/bounce_rules.spec.js

```diff
@@ -14,4 +14,25 @@ describe("Bounce Rules Page", () => {
     BounceRulesPage.ruleFilter.should("be.visible");
     BounceRulesPage.ruleTable.should("be.visible");
   });
+
+  it("should create a bounce rule", () => {
+    BounceRulesPage.createRuleButton.click();
```

Comment:
Ahh how bout we have a function called createBounceRule(bounceRule) { } inside the page object and pass in an object with all those parameters that encapsulates everything to create a bounce rule and then chain the last action of the createBounceRule with a .then and make the final assertions in there? fillForm is a bit too generic as there are other forms on the bounce rules page like the searching area and having too many parameters requires a developer to know the exact order of things when we can pass an object through the overall higher level createBounceRule function like bounceRule = { id: "1", ..., }